### PR TITLE
Mc us35

### PR DESCRIPTION
**Pull request by Mariana-21**
## All Submissions:

* [X] Merge the target branch into the PR branch. Fix any conflicts that might appear.
* [X] Have you checked to ensure that other pull request are approved for merge [Pull Requests](../../../pulls)?
* [ ] What stories does the PR include
userstory 35
### New Feature Submissions:

* [X] Have you added an explanation of what your changes do?
* [X] Have you lint your code locally prior to submission?

### Testing:

* [X] Have you written new tests for your core changes, as applicable?
* [X] Have you successfully ran tests with your changes locally?
* [X] All new and existing tests passed.

**Test coverage** = 100% model/features
--------------------------------------------------------------------------------
### Change Log

* Created new test in dashboard spec
* Created new order link in merchant index
* Created new many to many relationship between merchant and orders
* Created new test in model/orders spec

### CODE REVIEW

File: app/models/merchant.rb [hunk 1]
```diff
@@ -26,4 +27,8 @@ def distinct_cities
     item_orders.distinct.joins(:order).pluck(:city)
   end
 
+  def pending_orders
```
Comment (tylertomlinson) [inline app/models/merchant.rb:30]: Way to have this on the model level

File: spec/features/merchant/dashboard_spec.rb [hunk 2]
```diff
@@ -16,5 +16,46 @@
...
+# Each order listed includes the following information:
+
+
+
+
+
+# - the ID of the order, which is a link to the order show page ("/merchant/orders/15")
+# - the date the order was made
+# - the total quantity of my items in the order
+# - the total value of my items for that order
```
Comment (tylertomlinson) [inline spec/features/merchant/dashboard_spec.rb:61]: Maybe delete

**Review by tylertomlinson — APPROVED** [review]

File: app/models/merchant.rb [hunk 3]
```diff
@@ -2,6 +2,7 @@ class Merchant <ApplicationRecord
   has_many :items, dependent: :destroy
   has_many :item_orders, through: :items
   has_many :users
+  has_many :orders, through: :item_orders
```
Comment (alex-latham) [inline app/models/merchant.rb:5]: This was so tricky!

**Review by alex-latham — APPROVED** [review]
Looking good

**Pull request merged**